apps/prairielearn/src/pages/instructorAssessmentGroups/formDeleteGroup.ejs

```diff
@@ -1,4 +1,5 @@
 <form name="delete-group-form" method="POST">
+  <p>Are you sure you want to delete the group <strong><%= row.name %></strong>?</p>
```

Comment:
This popover looked weird without at least some text, so I added a short confirmation text.